src/common/ActionButtons/ActionButton.tsx

```diff
@@ -0,0 +1,85 @@
+import { FC } from "react";
+import classnames from "classnames";
+
+import { ActionButtonsProps } from "./ActionButtons";
+
+import { useMessageContext } from "../../hooks";
+import { getWebchatButtonLabel } from "../../utils";
+import { sanitizeHTML } from "../../sanitize";
+
+import classes from "./ActionButton.module.css";
+
+interface ActionButtonProps extends React.HTMLAttributes<HTMLDivElement> {
+	action: ActionButtonsProps["action"];
+	className?: string;
+	button: ActionButtonsProps["payload"][number];
+	total: number;
+	position: number;
+	disabled?: boolean;
+}
+
+/**
+ * Postback, phone number, and URL buttons
+ */
+const ActionButton: FC<ActionButtonProps> = props => {
+	const { button, total, position } = props;
+	const { config, onEmitAnalytics } = useMessageContext();
+
+	const buttonLabel = getWebchatButtonLabel(button) || "";
+	const __html = config?.settings?.disableHtmlContentSanitization
+		? buttonLabel
+		: sanitizeHTML(buttonLabel);
+	const buttonType = (button as any).type || (button as any).content_type;
+	const buttonPayload = button.payload;
+	const isPhoneNumber = buttonPayload && buttonType === "phone_number";
+	const buttonTitle = button.title || "";
+
+	const isWebURLButtonTargetBlank = (button as any).target !== "_self";
+	const isWebURL = buttonType === "web_url";
+
+	const buttonTitleWithTarget =
+		isWebURL && isWebURLButtonTargetBlank ? buttonTitle + "Opens in new tab" : button.title;
+	const ariaLabel =
+		total > 1
+			? `Item ${position} of ${total}: ${buttonTitleWithTarget}`
+			: buttonTitleWithTarget;
+
+	const A = (props: React.HTMLAttributes<HTMLAnchorElement>) => <a {...props} />;
```

Comment:
you can define here the "href", and remove from line 80. Doing like that, you will not need the "any" on line 50.

Reply:
fixed